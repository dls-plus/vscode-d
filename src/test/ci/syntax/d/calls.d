void main() {
	auto headerBar = new HeaderBar();
	headerBar.setTitle("Test");
	headerBar.setShowCloseButton(true);
	setTitlebar(headerBar);
}