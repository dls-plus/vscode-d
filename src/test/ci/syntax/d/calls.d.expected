[{"start":0,"end":4,"scope":"storage.type.basic-type.d"},{"start":4,"end":5,"scope":"source.d"},{"start":5,"end":9,"scope":"entity.name.function.d"},{"start":9,"end":10,"scope":"source.d"},{"start":10,"end":11,"scope":"source.d"},{"start":11,"end":14,"scope":"source.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":5,"scope":"storage.type.basic-type.d"},{"start":5,"end":6,"scope":"source.d"},{"start":6,"end":15,"scope":"variable.d"},{"start":15,"end":16,"scope":"source.d"},{"start":16,"end":17,"scope":"keyword.operator.assign.d"},{"start":17,"end":18,"scope":"source.d"},{"start":18,"end":22,"scope":"keyword.other.new.d"},{"start":22,"end":31,"scope":"entity.name.function.d"},{"start":31,"end":32,"scope":"source.d"},{"start":32,"end":33,"scope":"source.d"},{"start":33,"end":35,"scope":"meta.statement.end.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":19,"scope":"entity.name.function.d"},{"start":19,"end":20,"scope":"source.d"},{"start":20,"end":21,"scope":"string.double-quoted-string.d"},{"start":21,"end":25,"scope":"string.character.d"},{"start":25,"end":26,"scope":"string.double-quoted-string.d"},{"start":26,"end":27,"scope":"source.d"},{"start":27,"end":29,"scope":"meta.statement.end.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":29,"scope":"entity.name.function.d"},{"start":29,"end":30,"scope":"source.d"},{"start":30,"end":34,"scope":"constant.language.boolean.true.d"},{"start":34,"end":35,"scope":"source.d"},{"start":35,"end":37,"scope":"meta.statement.end.d"}]
[{"start":0,"end":1,"scope":"source.d"},{"start":1,"end":12,"scope":"entity.name.function.d"},{"start":12,"end":13,"scope":"source.d"},{"start":13,"end":22,"scope":"variable.d"},{"start":22,"end":23,"scope":"source.d"},{"start":23,"end":25,"scope":"meta.statement.end.d"}]
[{"start":0,"end":2,"scope":"source.d"}]